liaisons/opcf/liaisons-context.md

```diff
@@ -0,0 +1,44 @@
+# Organization Name
+OPC Foundation (https://opcfoundation.org/)
+
+## Purpose 
+OPC UA from OPCF is one of the prominent automation standards for device communication in manufacturing.
+To describe OPC UA-based devices with a W3C WoT Thing Description, specific metadata is required. This metadata provides 
+information that enables the connection to an OPC UA device/server and indicates how to retrieve specific OPC UA data and/or 
+functions from the device.
+
+The usage of those specific metadata in the WoT Thing Description should be described in an 'OPC UA Binding' (W3C Note) document, 
+similar to what exists, e.g., for Modbus.
+
+## Participants
+Who is involved and what is their role?
+* members from OPCF and/or WoT Connectivity WG (e.g., for OPC UA metadata support) 
+* members from W3C WoT (applying OPC UA metadata to WoT Thing Description)
+
+## Scope 
+The primary aim is to collaboratively define the aforementioned 'OPC UA Binding' document, which can be utilized to create the Thing Description for a device/server with an OPC UA interface. Doing this, OPC UA-specific metadata are required, which either need to be formally selected (e.g, for node addressing like with nodeId or extendedNodeId), or newly defined within an OPC UA-specific namespace (e.g., for security purposes). In addition, a strategy will be provided how Thing Descriptions can be generated automatically using OPC UA mechanisms, such as those from UA nodesets or UA browsing mechanisms.
```

Comment:
should "browsing" be "discovery" or "browsing and discovery"?

Reply:
not sure what terminology they officially use, I have heard both

Reply:
UA use the official "browse" term https://reference.opcfoundation.org/Core/Part4/v104/docs/5.8.2